dist/bson.bundle.js

```diff
@@ -3002,6 +3002,225 @@ var timestamp = Timestamp;
 var Timestamp_1 = Timestamp;
 timestamp.Timestamp = Timestamp_1;
 
+var global$1 = typeof global !== "undefined" ? global : typeof self !== "undefined" ? self : typeof window !== "undefined" ? window : {};
```

Comment:
I wouldn't expect these to show up or be wanted by users in this bundle file. Is it a result of testing?